cuda_core/tests/requirements-cu11.txt

```diff
@@ -2,3 +2,4 @@ pytest
 # TODO: remove this hack once cupy has a cp313 build
 cupy-cuda11x; python_version < "3.13"
 nvidia-cuda-runtime-cu11  # headers consumed by CuPy
+torch # For PyTorch example
```

Comment:
The pytorch website indicates this installs a CUDA 12.6 based version on Linux? We also need to make sure this works for Windows.

Reply:
Yeah windows would be CPU only builds by default

I can add a reference to the pytorch index to the right binaries